### PR TITLE
Article Refresh/New Nav:  more subnav tweaks

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

Three things:
* adding campaign codes to the footer https://github.com/guardian/frontend/commit/768de74503b783bd4364e812812c80e1daf224b2
* adding the parentSection on tertiary nav lists https://github.com/guardian/frontend/commit/a6a7670dafeb01d7656752a5897bd77f84635658 for example football here:
image
* ensuring the primary links (news, life, arts, sports, opinion) don't show up in the subnav, because they are already visible.
**Before:**
image

**After:**
image

cc @guardian/dotcom-platform 

## What is the value of this and can you measure success?
campaign codes are good for tracking clicks. Other things are for user experience reasons.

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Tested in CODE?
Nope

### CODE REVIEW

File: common/app/common/NewNavigation.scala [hunk 1]
```diff
@@ -223,41 +216,34 @@ object NewNavigation {
...
+      dating.copy(dating.url + "?soulmates_uk_web_newheader"),
       NavLink("professional", "/guardian-professional"),
-      NavLink("masterclasses", "/guardian-masterclasses?INTCMP=NGW_TOPNAV_UK_GU_MASTERCLASSES"),
+      masterClasses.copy(masterClasses.url + "?masterclasses_uk_web_newheader"),
       NavLink("courses", "/?INTCMP=NGW_TOPNAV_UK_GU_COURSES"),
       NavLink("holidays", "https://holidays.theguardian.com/?utm_source=theguardian&utm_medium=guardian-links&utm_campaign=topnav&INTCMP=topnav"),
-      NavLink("today's paper", "/theguardian"),
-      NavLink("the observer", "/observer"),
-      NavLink("crosswords", "/crosswords")
+      todaysPaper, observer, crosswords
```
Comment (jfsoul) [inline common/app/common/NewNavigation.scala]: Minor niggle: I prefer these to be on their own lines but that is personal preference (applies elsewhere too)

File: common/app/common/NewNavigation.scala [hunk 2]
```diff
@@ -223,41 +216,34 @@ object NewNavigation {
     val name = ""
 
     val uk = NavLinkLists(List(
-      NavLink("apps", "/global/ng-interactive/2014/may/29/-sp-the-guardian-app-for-ios-and-android"),
-      NavLink("jobs", "https://jobs.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_JOBS"),
-      NavLink("dating", "https://soulmates.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_SOULMATES"),
+      apps.copy(apps.url + "?apps_uk_web_newheader"),
```
Comment (jfsoul) [inline common/app/common/NewNavigation.scala]: does this need to be `apps.copy(url = apps.url + "?apps_uk_web_newheader")`?

Comment (jfsoul) [inline common/app/common/NewNavigation.scala]: I thought it would overwrite `title` with this, maybe you can explain why if it doesn't 😄

Comment (NataliaLKB) [inline common/app/common/NewNavigation.scala]: Haha! This is a result of me trying to multitask and doing it unsuccessfully. Fixed now.

File: common/app/common/NewNavigation.scala [hunk 3]
```diff
@@ -223,41 +216,34 @@ object NewNavigation {
...
     val int = NavLinkLists(List(
-      NavLink("apps", "/global/ng-interactive/2014/may/29/-sp-the-guardian-app-for-ios-and-android"),
-      NavLink("dating", "https://soulmates.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_SOULMATES"),
-      NavLink("jobs", "https://jobs.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_JOBS"),
-      NavLink("masterclasses", "/guardian-masterclasses?INTCMP=NGW_TOPNAV_UK_GU_MASTERCLASSES"),
-      NavLink("today's paper", "/theguardian"),
-      NavLink("the observer", "/observer"),
-      NavLink("crosswords", "/crosswords"),
-      NavLink("video", "/video")
+      apps.copy(url = apps.url + "?INTCMP=apps_int_web_newheader"),
```
Comment (NataliaLKB) [inline common/app/common/NewNavigation.scala]: @gtrufitt thanks for the pointer! Yes they want `apps_int_web_newheader` to be the value of `INTCMP`

**Review by jfsoul — APPROVED** [review]
Sweet! 🍬

**Comment by NataliaLKB** [comment]
@jfsoul Thank you!!! 🌟

**Pull request merged**